BackofficeBundle/Resources/public/coffee/FullPageFormView.coffee

```diff
@@ -2,21 +2,21 @@ FullPageFormView = OrchestraView.extend(
   el: '#content'
 
   initialize: (options) ->
+    @initializer options
```

Comment:
I'm actually not impacted (I'm not overloading the initilize method yet).
However I don't see the point (actually I do for FullPagePanelView, but... : ) of delegating the init work in another method while not making the initialize() really generic.
The loadtemplates should, in my opinion, be called in the OrchestraView and provided with an array of templates that would therefore become a property of the view. Therefore you no more need to manually call the initializer and loadtemplates in the children methods.